.github/workflows/main-genstudio-create-validation.yaml

```diff
@@ -0,0 +1,17 @@
+name: Create Validation Main
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "genstudio-create-validation/**"
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    uses: ./.github/workflows/aio-app-template.yml
+    with:
+      app: genstudio-create-validation
+      environment: '["Genstudio Engineering 01 - genstudio-create-validation - workspace Production", "GenStudio GA PAT04 - genstudio-create-validation - workspace Production"]'
+    secrets: inherit
```

Comment:
nit: can we mention in readme that we are not using the workflows inside the apps but that those can be used by individual projects to automate deployment. we just built a custom way because we deploy to many different envs

Reply:
updated